lib/msal-common/src/client/RefreshTokenClient.ts

```diff
@@ -59,7 +62,32 @@ export class RefreshTokenClient extends BaseClient {
             undefined,
             true,
             request.forceCache
-        );
+        ).then((result: AuthenticationResult) => {
+            if (response.body?.refresh_token) {
```

Comment:
@jasonnutter  Yes, this if-else block is to differentiate between when there is no refresh token in the response (i.e no Refresh token is returned at all) vs when there is a refresh token of length 0 (e.g due to network error, or something else). 

It was desired to make a distinction between these two cases, https://github.com/AzureAD/microsoft-authentication-library-for-js/pull/5063#discussion_r960918678

Reply:
I disagree that this is something we should differentiate. Comment left below.

Reply:
Agree, addressed in latest changes .